examples/pipelines/windows-customize/README.md

```diff
@@ -1,11 +1,15 @@
-# Windows 10 Customize Pipeline
+# Windows Customize Pipeline
 
-This pipeline clones the DataVolume of a basic and generalized Windows 10 installation and runs arbitrary customization
+This pipeline clones the DataVolume of a basic and generalized Windows 10/11/2k22 installation and runs arbitrary customization
 commands through an unattend.xml after startup  of the VM. As an example a ConfigMap which installs Microsoft SQL
-Server Express and generalizes the VM after is included (`windows10-sqlserver`). For basic setup after the first start
-of a customized VM an example unattend.xml is included in the pipeline's ConfigMap `windows10-unattend`.
+Server Express and generalizes the VM after is included (e.g. `windows10-sqlserver`, or `windows11-vs-code`). For basic setup after the first start
+of a customized VM an example unattend.xml is included in the pipeline's ConfigMap `windows10-unattend`, or `windows11-vs-code`.
 
-The provided reference ConfigMap (`windows10-sqlserver`) boots Windows 10 into Audit mode, applies the customizations as
+This example pipeline can be used for running windows 10, 11 or 2k22 (or others - not tested!). Always adjust pipeline parameters 
+for windows version you are currently using(e.g. different template name, different base image name, ...). It is possible to use 
+`windows10-unattend` config map for win11/2k22 and vice versa (`windows11-vs-code` for win10/2k22).
```

Comment:
Is it worth mentioning this or why is there a need for different config maps at all then?